spec/index.bs

```diff
@@ -1929,6 +1931,29 @@ interface GPUAdapterInfo {
         this value is not recommended. Applications which change their behavior based on the
         {{GPUAdapterInfo}}, such as applying workarounds for known driver issues, should rely on the
         other fields when possible.
+
+    : <dfn>properties</dfn>
```

Comment:
Is there a need/purpose for encapsulating these things in a new GPUAdapterProperties object?

Can the two new fields be included directly in GPUAdapterInfo?

Somewhat related in C-style APIs, I expect we would have a new chained struct, e.g. either GPUAdapterSubgroupInfo  (or GPUAdapterPropertiesSubgroupInfo if we do introduce a new GPUAdapterProperties)